jgiven-core/src/main/java/com/tngtech/jgiven/impl/util/ReflectionUtil.java

```diff
@@ -1,14 +1,13 @@
 package com.tngtech.jgiven.impl.util;
 
+import static java.lang.String.format;
```

Comment:
protip: separate formatting changes from functional changes.  (at least in a separate commit), especially if they cover the whole file.  This eases the reviewing process.